### DISCUSSION TITLE
ESP32 Hardware

### DISCUSSION THREAD

**Discussion by prasannabe2004** [Mair/esp32-course#92] [Course Qeustions] [answered]
HI, I just started with the course.. I am having difficulty in finding the ESP32 wroom devkit v1 in aliexpress due to high lead time. in shipping Any alternative vendor to purchase ESP32 wroom devkit v1 board for this course preferably in United States.

**Comment by rprr** [comment, marked as answer]
What about  ebay?  Or amazon?

**Reply by Mair** [reply]
Those are also good
here are 3 for under $20 that's under $7.00 / unit which is not bad on [amazon](https://www.amazon.com/Development-Microcontroller-Integrated-Antenna-Amplifiers/dp/B09GK74F7N/ref=sr_1_5?keywords=esp32+development+board&pd_rd_r=a40cc1f8-260b-4926-9fc1-cab4601ae658&pd_rd_w=0YNdL&pd_rd_wg=3a2Rk&pf_rd_p=4fa0e97a-13a4-491b-a127-133a554b4da3&pf_rd_r=ERCVTQYW1F8FJ0HV9192&qid=1651113652&sr=8-5)

**Reply by prasannabe2004** [reply]
Thank you. I purchased this from amazon.
